workers/ohsome_quality_analyst/indicators/mapping_saturation/indicator.py

```diff
@@ -150,37 +132,33 @@ def calculate(self) -> bool:
             #    growth should be 0.0
             self.growth = 1 - self.saturation
 
+            description = Template(self.metadata.result_description).substitute(
+                saturation=self.saturation,
+                growth=self.growth,
+            )
+            if self.saturation == 0:
+                self.result.label = "red"
+                self.result.value = 0.0
+                self.result.description = (
+                    description + self.metadata.label_description["red"]
+                )
             # growth is larger than 3% within last 3 years
-            if self.growth <= THRESHOLD_YELLOW:
-                label = "green"
-                value = 1.0
-                description += self.metadata.label_description["green"]
+            elif self.growth <= THRESHOLD_YELLOW:
+                self.result.label = "green"
+                self.result.value = 1.0
+                self.result.description = (
+                    description + self.metadata.label_description["green"]
+                )
+            # growth level is better than the red threshould
             else:
-                # growth level is better than the red threshould
-                label = "yellow"
-                value = 0.5
-                description += self.metadata.label_description["yellow"]
-            description_template = Template(
-                self.metadata.result_description
-            ).substitute(saturation=self.saturation, growth=self.growth)
-            description = description_template + " " + description
-            self.result.label = label
-            self.result.value = value
-            self.result.description = description
-            logging.debug(
-                f"Saturation result value: {self.saturation}, label: {label},"
-                f" value: {value}, description: {description}"
-            )
+                self.result.label = "yellow"
+                self.result.value = 0.5
+                self.result.description = (
+                    description + self.metadata.label_description["yellow"]
+                )
+            return True
         else:
-            # no data / deletion of all data
-            label = "undefined"
-            value = None
-            description += self.metadata.label_description["undefined"]
-            self.result.label = label
-            self.result.value = value
-            self.result.description = description
```

Comment:
This isn't necessary anymore, because you use the default values of the `BaseIndicator`?

Reply:
Yes